.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Discord yells at us if this file gets uploaded
-token.txt
+env.json
```

Comment:
This file should not be ignored as it is required to be filled out for the bot to run. You should only check in this file if you change the keys inside of it, otherwise discard the changes before pushing.